### PR TITLE
Release SDK bump and SSR upgrade to prod

### PR DESCRIPTION
**Pull request by BowTiedHeron**

### CODE REVIEW

File: components/HoneyTable/HoneyTable.css.ts [hunk 1]
```diff
@@ -45,6 +45,10 @@ export const honeyTableRow = style({
   cursor: 'pointer'
 });
 
+export const loadingCursor = style({
```
Comment (BowTiedHeron) [inline components/HoneyTable/HoneyTable.css.ts:48]: reconsider loading animation as discussed with @tomjpandolfi

Comment (BowTiedHeron) [reply components/HoneyTable/HoneyTable.css.ts:48]: cc: @oshioked

File: components/HoneyTable/HoneyTable.tsx [hunk 2]
```diff
@@ -40,6 +42,9 @@ const HoneyTable: FC<HoneyTableProps> = props => {
     if (hasRowsShadow && selectedRowsKeys.includes(key)) {
       classes.push(styles.honeyTableRowSelected);
     }
+    if (isLoading) {
+      classes.push(styles.loadingCursor);
```
Comment (BowTiedHeron) [inline components/HoneyTable/HoneyTable.tsx:46]: re- loading comment

File: helpers/format.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import BN from 'bn.js';
```
Comment (BowTiedHeron) [inline helpers/format.ts:2]: check if BN is used

File: contexts/AccountsProvider.tsx [hunk 4]
```diff
@@ -124,25 +124,47 @@ export const AccountsProvider: React.FC<React.ReactNode> = ({ children }) => {
   );
 };
 
-async function getNFTImgURI(uri: string) {
+const allowedCollections = [
```
Comment (BowTiedHeron) [inline contexts/AccountsProvider.tsx:127]: create issue to remove this and migrate to stand alone governance branch/repo

File: helpers/marketHelpers/index.tsx [hunk 5]
```diff
@@ -332,7 +262,102 @@ export const marketCollections: MarketTableRow[] = [
...
+  //     verifiedCreator: 'EZcZTsLpvdisPgQy5TcCALYgmGKRMNvvyJ89t1LFWmum',
+  //     marketName: 'Trippin ape tribe',
+  //     marketImage:
+  //       'https://img-cdn.magiceden.dev/rs:fill:400:400:0:0/plain/https://i.imgur.com/iFgvQva.png',
+  //     discountedMarket: false
+  //   }
+  // },
 ];
 
 // fees
```
Comment (BowTiedHeron) [inline helpers/marketHelpers/index.tsx:363]: validate if the below is still being used

File: pages/borrow/index.tsx [hunk 6]
```diff
@@ -835,14 +996,13 @@ const Markets: NextPage = () => {
     try {
       if (!mintID) return;
       toast.processing();
-      // loop through collections -> verified creator is used for validation
+      // TODO: set verified creator of active market along with currentMarketId
```
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:999]: is there an issue for this? otherwise best to create one to not forget

File: pages/borrow/index.tsx [hunk 7]
```diff
@@ -954,16 +1117,19 @@ const Markets: NextPage = () => {
   async function executeBorrow(val: any, toast: ToastProps['toast']) {
     try {
       if (!val) return toast.error('Please provide a value');
-      // if (val == 1.6) val = val - 0.01;
+      // TODO: make the token mint dynamic by importing it from marketcollection / fetchAllMarkets
```
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:1120]: create issue

File: pages/borrow/index.tsx [hunk 8]
```diff
@@ -1017,15 +1178,19 @@ const Markets: NextPage = () => {
   async function executeRepay(val: any, toast: ToastProps['toast']) {
     try {
       if (!val) return toast.error('Please provide a value');
+      // add additional value if user wants to repay 100% of loan due to interest rate not being included
+      if (val == userDebt) val += 0.1;
+      // TODO: make dynamic - marketCollections or fetchAllMarkets
```
Comment (BowTiedHeron) [inline pages/borrow/index.tsx:1183]: create issue/add to issue

**Review by BowTiedHeron — COMMENTED** [review]
Initial review. Create issues for comments and continue with merge strat so others can continue their work.

**Review by bowtiedpyro — APPROVED** [review]
Looks good - comments will be noted / worked on, no urgent fixes for now

**Pull request merged**